### PR TITLE
Always encode strings in Decoder.set_property

### PR DESCRIPTION
**Pull request by will-moore**
Required to fix https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/31/testReport/OmeroWeb.test.integration.test_api_containers/TestContainers/test_container_crud_Project_/
along with https://github.com/ome/omero-web/pull/38

### CODE REVIEW

**Comment by will-moore** [comment]
Travis failing with ```ImportError: No module named past.builtins```

**Comment by sbesson** [comment]
This probably means `future` also needs to be added to the dependencies of `omero-marshal`

File: omero_marshal/decode/__init__.py [hunk 1]
```diff
@@ -34,7 +35,8 @@ def to_unit(self, v):
 
     def set_property(self, target, prop, value):
         field_info = getattr(target._field_info, prop)
-        if isinstance(value, unicode):
+        # When would we NOT want to do this?
```
Comment (joshmoore) [inline omero_marshal/decode/__init__.py]: There's at least one other example of this, but I'll have to dig it up. I think this is a case where one differentiates between py2/py3.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-python-superbuild-push#58](https://idr1-slot3.openmicroscopy.org:48443/jenkins/job/OMERO-python-superbuild-push/58/). See the [console output](https://idr1-slot3.openmicroscopy.org:48443/jenkins/job/OMERO-python-superbuild-push/58/consoleText) for more details.
Possible conflicts:
  - PR #64 manics 'import omero.all to avoid ordering errors'
    - omero_marshal/decode/__init__.py

--conflicts

**Comment by joshmoore** [comment]
@will-moore : you're proposing release as 0.6.3?

**Comment by will-moore** [comment]
Sounds good, yes.

**Pull request merged**